packages/data-context/src/sources/VersionsDataSource.ts

```diff
@@ -161,45 +160,43 @@ export class VersionsDataSource {
       }
     }
 
-    try {
-      const projectPath = this.ctx.currentProject
-
-      if (projectPath) {
-        const dependenciesToCheck = WIZARD_DEPENDENCIES
-
-        debug('Checking %d dependencies in project', dependenciesToCheck.length)
-        // Check all dependencies of interest in parallel
-        const dependencyResults = await Promise.allSettled(
-          dependenciesToCheck.map(async (dependency) => {
-            const result = await this.ctx.util.isDependencyInstalled(dependency, projectPath)
-
-            // If a dependency isn't satisfied then we are no longer interested in it,
-            // exclude from further processing by rejecting promise
-            if (!result.satisfied) {
-              throw new Error('Unsatisfied dependency')
-            }
-
-            // We only want major version, fallback to `-1` if we couldn't detect version
-            const majorVersion = result.detectedVersion ? semver.major(result.detectedVersion) : -1
-
-            // For any satisfied dependencies, build a `package@version` string
-            return `${result.dependency.package}@${majorVersion}`
-          }),
-        )
-        // Take any dependencies that were found and combine into comma-separated string
-        const headerValue = dependencyResults
-        .filter(this.isFulfilled)
-        .map((result) => result.value)
-        .join(',')
-
-        if (headerValue) {
-          manifestHeaders['x-dependencies'] = headerValue
+    if (this._initialLaunch) {
```

Comment:
I'm a little hazy on `initialLaunch` behavior. I get the intent is to capture this data just once on initial launch of Cypress, but how does this behave with global mode?

Reply:
Good catch - as a result of this, if a project is opened via global mode, we won't capture any dependencies. Spoke with @marktnoonan about it, and he updated the requirements to note this behavior.